src/typescript/azure-openapi-validator/rules/utilities/rules-helper.ts

```diff
@@ -39,4 +39,9 @@ function getResponseSchema(response: Object, doc): any {
     return schemaProperties;
   }
   return schema.properties;
+}
+
+export function IsValidOperation(operation: string): boolean {
+  let validOperations = ["put", "get", "patch", "post", "head", "options", "delete"]
```

Comment:
what if the operation is not valid? Any error returned to the user?

Reply:
it just make sure the rule don't crash  , there are other rules to check this